charts/openshift-metering/templates/reporting-operator/reporting-operator-aws-credentials-secrets.yaml

```diff
@@ -3,7 +3,7 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: reporting-operator-aws-credentials-secrets
+  name: reporting-operator-aws-credentials
```

Comment:
I like this better - I'll have to update the Presto and reporting-operator (api, metrics, presto) to remove the `-secrets` suffix from the secret template name.

Reply:
Yeah, I basically changed it so the name of the secret created isn't controllable by the user. If you look at the deployment you'll see how I use `default` to go to use this secret name if the user didn't specify an `awsCredentialSecretName`.